utility/lambdas/ecs-asg-sizer/lambda_function.py

```diff
@@ -0,0 +1,115 @@
+import boto3
```

Comment:
Make sure this is following the newish linting style; details in Contributing.md